plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/converter/GlueToTrinoConverter.java

```diff
@@ -107,9 +108,12 @@ public static Table convertTable(com.amazonaws.services.glue.model.Table glueTab
 
         StorageDescriptor sd = glueTable.getStorageDescriptor();
 
-        if (isIcebergTable(tableParameters) || (sd == null && isDeltaLakeTable(tableParameters))) {
+        if (isIcebergTable(tableParameters) ||
+                (sd == null && isDeltaLakeTable(tableParameters)) ||
+                (sd == null && isTrinoMaterializedView(getTableType(glueTable), glueTable.getParameters()))) {
```

Comment:
`glueTable.getParameters()`  may be null and cause NPE inside the called function

use
```
firstNonNull(glueTable.getParameters(), Map.of())
```